docs/src/prototypes/popups/GridImagePicker/GridImagePickerItem.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react'
+import { Image, Button } from '@stardust-ui/react'
+import * as _ from 'lodash'
+
+export interface GridPickerItemProps {
+  as?: keyof React.ReactHTML
+  title?: string
+  imageSrc: string
+  onClick?: (e) => void
+}
+
+const imageStyles = {
+  width: '100%',
+}
+
+const imageButtonStyles = {
+  minWidth: '56px',
+  height: '56px',
+  padding: '0',
+  background: '#fff',
+}
+
+class GridImagePickerItem extends React.Component<GridPickerItemProps, any> {
+  static defaultProps = {
+    as: 'li',
+  }
+
+  handleClick = e => {
+    if (e.target) {
+      const btn = e.target.nodeName !== 'IMG' ? e.target : e.target.parentElement
+      const selectedItem = btn && btn.title
+      selectedItem && alert(`Selected item is: ${selectedItem}`)
+    }
```

Comment:
May be we can pass something as `data` attribute?

```jsx
handleClick = (e, props) => {
  const title = props['data-title']
  // Other goes there
}

render() {
  return <Button data-title='COOL TITLE' />
}
```

Reply:
```title``` attribute is needed because that is read by screen reader. And ```handleSelection``` it is just to show that something happened when you click or press Enter / Space on an image.

Reply:
It was stupid idea at all.

Why we can't just do this?

```diff
-  handleClick = e => {
-    if (e.target) {
-      const btn = e.target.nodeName !== 'IMG' ? e.target : e.target.parentElement
-      const selectedItem = btn && btn.title
-      selectedItem && alert(`Selected item is: ${selectedItem}`)
-    }
+ handleClick = (e, props) => {
+   const { title } = props
+
+   alert(`Selected item is: ${title}`)
+ }
```

We are passing a `title` prop to `Button`, so we will get it in `onClick`.